### ISSUE TITLE
508-defect-3 [COGNITION]: Find helpful VA phone number link should be styled as a link

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#27594]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## Details

"Find helpful VA phone numbers" is _styled_ as a button but is written as a `link` which is materially dishonest.
- Keyboard users may become frustrated that they cannot activate it using `space` which is what a button typically affords.
- Sighted screen reader users may be confused that it appears within the links list and not the buttons list. 
- Voice AT users may be unable to activate it when they issue the command as a "button" instead of a "link."

For more on this long-tail effort, check out our [buttons vs. links Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1610052905994/253723ef4b2ee9a726f538876a257df94d468db1) and this [Slack thread discussing _why_ material honesty is so important](https://dsva.slack.com/archives/C8E985R32/p1610659601009200?thread_ts=1610652897.002600&cid=C8E985R32).

"A few years at an accessibility event, a blind man was telling a story about how he was on the phone with someone who was trying to help him sign up for a service. The person on the phone told him to click the “sign up button” and he couldn’t find it because it was a link. He told the story with a lot more gravity than I can, but that was the point where I realized how important the human-human communication part of accessibility is, it’s not just machine to human." - Tim Wright

## Acceptance Criteria

- [ ] "Find a helpful VA phone number" is styled as a link

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution (if known)

Use a link or consider using the [VA Design System's experimental action link.](https://design.va.gov/experimental-design/action_links)

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [VA Design System guidance on buttons](https://design.va.gov/components/buttons#guidance)

## Screenshots or Trace Logs

<img width="1284" alt="Screen Shot 2021-07-20 at 10 52 47 AM" src="https://user-images.githubusercontent.com/14154792/126349982-42790a22-043d-41d9-b95a-1428fe06dbcf.png">

**Comment by laflannery** [comment]
This is resolved and is no longer styled as a button.  Will close in 1 week unless told otherwise
